pkg/controllermanager/controller/project/project_control.go

```diff
@@ -46,6 +46,19 @@ func (c *Controller) projectUpdate(oldObj, newObj interface{}) {
 	if !ok {
 		return
 	}
+	oldProject, ok := oldObj.(*gardencorev1beta1.Project)
+	if !ok {
+		return
+	}
+
+	if newProject.Status.LastActivityTimestamp != oldProject.Status.LastActivityTimestamp {
```

Comment:
```suggestion
	if reflect.DeepEqual(newProject.Status.LastActivityTimestamp, oldProject.Status.LastActivityTimestamp) {
```
The pointer comparison before will always be true.